From a33c09fbc7327c05407d83d62df9ebd8daf6629f Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Mon, 7 Nov 2011 11:26:19 +0800
Subject: [PATCH 0572/2463] ENGR00161510 [MX6]Fix build warning

arch/arm/mach-mx6/cpu.c:39: warning:
'arm_base' defined but not used.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/mach-mx6/cpu.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu.c b/arch/arm/mach-mx6/cpu.c
index 0833b57..f2b9345 100644
--- a/arch/arm/mach-mx6/cpu.c
+++ b/arch/arm/mach-mx6/cpu.c
@@ -36,7 +36,6 @@ extern void mx6_wait(void);
 
 
 struct cpu_op *(*get_cpu_op)(int *op);
-static void __iomem *arm_base = IO_ADDRESS(MX6Q_A9_PLATFRM_BASE);
 bool enable_wait_mode;
 
 void __iomem *gpc_base;
-- 
1.7.7.4

